### PR TITLE
Delete 23 test for swapping.py

### PR DESCRIPTION
**Pull request by itskumarsuraj**
please delete

### CODE REVIEW

**Comment by gourav47** [comment]
Thanks

**Comment by itskumarsuraj** [comment]
Your welcome

File: 23 test for swapping.py [hunk 1]
```diff
@@ -1,3 +0,0 @@
-a=20
-b=45
```
Comment (itskumarsuraj) [inline 23 test for swapping.py:2]: Haa Hsjsksndnd

**Review by itskumarsuraj — COMMENTED** [review]
Bejsj

File: 23 test for swapping.py (hunk 1 above)
Comment (itskumarsuraj) [reply 23 test for swapping.py:2]: Please check the code there seems to be some bug in it

**Comment by gourav47** [comment]
Let's check

**Comment by itskumarsuraj** [comment]
Sure

**Pull request closed without merging**